### ISSUE TITLE
Logging in Jenkins

### ISSUE THREAD

**Issue by magicmonty** [nuke-build/nuke#412]
**Update: Removed the End of task requests, as this is a different feature, which does not belong here ;)**

### Description
On BuildServers (in my case Jenkins) the output of the Nuke build log is quite ugly and hard to parse:

![grafik](https://user-images.githubusercontent.com/315639/75135565-36310000-56e2-11ea-8d91-3f78c3c6cf6c.png)

and

![grafik](https://user-images.githubusercontent.com/315639/75135673-9a53c400-56e2-11ea-95d8-228a9914348e.png)

It would be nice, if there would be a command line option where one can enable a simpler logging for the Tasks.

Proposal for the Task Start: 

```
---- Starting Task Clean ---- 
```

### Alternatives
Start:
```
+----------
| Clean
+--
```

or 
```
+==========
| Clean
+==
```

Another alternative could be to just implement this as a special output sink for Jenkins

**Comment by magicmonty** [comment]
I will provide a PR for this ;)

**Comment by magicmonty** [comment]
I changed this issue to Logging in Jenkins, because this was in fact two feature requests.

**Comment by matkoch** [comment]
can you try this please? https://medium.com/pacroy/how-to-fix-jenkins-console-log-encoding-issue-on-windows-a1f4b26e0db4

<blockquote><img src="https://miro.medium.com/max/1200/1*unw_OAKk9mV1EEhiNovhGg.jpeg" width="48" align="right"><div><img src="https://cdn-static-1.medium.com/_/fp/icons/favicon-rebrand-medium.3Y6xpZ-0FSdWDnPM3hSBIA.ico" height="14"> Medium</div><div><strong><a href="https://medium.com/pacroy/how-to-fix-jenkins-console-log-encoding-issue-on-windows-a1f4b26e0db4">How to Fix Jenkins Console Log Encoding Issue on Windows</a></strong></div><div>If you get alien characters in console log of your job like this:</div></blockquote>

**Comment by magicmonty** [comment]
Unfortunately this doesn't work for me. I tried both `-Dfile.encoding=UTF8` and `-Dfile.encoding=UTF-8` variants.

**Comment by magicmonty** [comment]
Now it shows 
![grafik](https://user-images.githubusercontent.com/315639/75158026-9b521900-5715-11ea-933c-a3d4cad6bce7.png)

**Comment by matkoch** [comment]
How are you invoking the build?

**Comment by magicmonty** [comment]
with the dotnet global tool (`dotnet nuke`) and as Pipeline over a Jenkinsfile

**Comment by matkoch** [comment]
Try invoking the bash or powershell script. It doesn’t make much sense to install the global tool

**Comment by magicmonty** [comment]
Ok nice to know, that worked fine. I think, that resolves it

**Comment by magicmonty** [comment]
Thanks for the help

**Comment by matkoch** [comment]
@magicmonty is it fixed only by changing the invocation, or do you still need to define the encoding?
